torch/_dynamo/variables/tensor.py

```diff
@@ -707,18 +711,57 @@ def var_getattr(self, tx, name):
             result = wrap_fx_proxy_cls(
                 target_cls=NumpyNdarrayVariable,
                 tx=tx,
-                proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
+                proxy=tx.output.create_proxy(
+                    "call_function",
+                    attr_wrapper,
+                    (self.as_proxy(), name),
+                    {},
+                ),
                 example_value=None,
                 **options,
             )
-        elif name in ["size", "itemsize", "strides", "shape", "ndim"]:
+        elif name in ["ndim", "itemsize", "shape"]:
             result = wrap_fx_proxy_cls(
                 target_cls=ConstantVariable,
                 tx=tx,
                 proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
                 example_value=None,
                 **options,
             )
+        elif name == "shape":
+            result = wrap_fx_proxy_cls(
+                target_cls=TupleVariable,
+                tx=tx,
+                proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
+                example_value=None,
+                **options,
+            )
+        elif name == "size":
+            result = wrap_fx_proxy_cls(
+                target_cls=ConstantVariable,
+                tx=tx,
+                proxy=tx.output.create_proxy(
+                    "call_method",
+                    "numel",
+                    (self.as_proxy(),),
+                    {},
+                ),
+                example_value=None,
+                **options,
+            )
+        elif name == "strides":
+            result = wrap_fx_proxy_cls(
+                target_cls=ConstantVariable,
+                tx=tx,
+                proxy=tx.output.create_proxy(
+                    "call_function",
+                    attr_wrapper,
+                    (self.as_proxy(), name),
```

Comment:
I think it will be much better if, instead of reusing the name var as is, you redefine the string literal here, emphasizing the fact that there is no *intrinsic* relationship between Numpy's attribute name, and PyTorch's function name. Case in point here, PyTorch tensor doesn't have a strides() method, it has a stride() (no s method). So this doesn't work. You probably need a test for this too.